mathics/builtin/patterns/composite.py

```diff
@@ -173,6 +173,11 @@ def match(self, expression: Expression, pattern_context: dict):
         #     yield new_vars_dict, rest
         self.pattern.match(expression, pattern_context)
 
+    def get_sort_key(self, pattern_sort=True):
```

Comment:
This change seems to be different than all of the other changes. Let's separate this part from the stuff that adds `"Pattern[expression, ... ` everywhere. 

Thanks.